go/vt/vttablet/tmclient/rpc_client_api.go

```diff
@@ -187,7 +187,7 @@ type TabletManagerClient interface {
 
 	// StopReplicationAndGetStatus stops replication and returns the
 	// current position.
-	StopReplicationAndGetStatus(ctx context.Context, tablet *topodatapb.Tablet) (*replicationdatapb.Status, error)
+	StopReplicationAndGetStatus(ctx context.Context, tablet *topodatapb.Tablet, stopIOThreadOnly bool) (*replicationdatapb.Status, *replicationdatapb.StopSlaveStatus, error)
```

Comment:
We want to move away from `slave` and use `replica` wherever possible. So maybe rename `StopSlaveStatus` to `StopReplicationStatus`.

Reply:
Done.